Products/CMFPlone/controlpanel/browser/relations_inspect.pt

```diff
@@ -41,7 +41,7 @@
            class="portalMessage warning alert alert-warning" role="alert"
            i18n:translate="">
              <strong>Warning!</strong> You have broken relations!
-             You can <a href="@@rebuild_relations">inspect and rebuild all relations</a>.
+             You can <a href="@@rebuild-relations">inspect and rebuild all relations</a>.
```

Comment:
In case you want to have the URL absolute (like we usually have):
```suggestion
             You can <a href="${here/portal_url}/@@rebuild-relations">inspect and rebuild all relations</a>.
```
but this is not really relevant for the issue you are addressing, so feel free to totally ignore this comment :)

Reply:
@ale-rt 
i have update the template with the absolute portal_url as you suggested and i added a test for check this. maybe a bit exaggerated, but like the docs say: add a test for every change ;-)